### PR TITLE
Platform Engineering guide: 7 steps

### PR DESCRIPTION
**Pull request by SaraDPH**

### CODE REVIEW

File: content/blog/the-guide-platform-engineering-idp-steps-best-practices/index.md [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+title: "The Guide to Platform Engineering: 7 Steps to Get It Right"
+allow_long_title: true
+
+# The date represents the post's publish date, and by default corresponds with
+# the date and time this file was generated. Dates are used for display and
+# ordering purposes only; they have no effect on whether or when a post is
+# published. To influence the ordering of posts published on the same date, use
+# the time portion of the date value; posts are sorted in descending order by
+# date/time.
+date: 2024-09-02T07:47:03Z
```
Comment (SaraDPH) [inline content/blog/the-guide-platform-engineering-idp-steps-best-practices/index.md:11]:
```suggestion
date: 2024-10-02T07:47:03Z
```

**Review by thtmnisamnstr — APPROVED** [review]

File: content/blog/the-guide-platform-engineering-idp-steps-best-practices/meta.png
Comment (dirien) [inline content/blog/the-guide-platform-engineering-idp-steps-best-practices/meta.png:1]: Is this okay to be the default image?

File: content/blog/the-guide-platform-engineering-idp-steps-best-practices/index.md [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+title: "The Guide to Platform Engineering: 7 Steps to Get It Right"
+allow_long_title: true
+
+# The date represents the post's publish date, and by default corresponds with
+# the date and time this file was generated. Dates are used for display and
+# ordering purposes only; they have no effect on whether or when a post is
+# published. To influence the ordering of posts published on the same date, use
+# the time portion of the date value; posts are sorted in descending order by
+# date/time.
+date: 2024-10-02T07:47:03Z
```
Comment (SaraDPH) [inline content/blog/the-guide-platform-engineering-idp-steps-best-practices/index.md:11]:
```suggestion
date: 2024-10-22T07:47:03Z
```

File: content/blog/the-guide-platform-engineering-idp-steps-best-practices/index.md [hunk 3]
```diff
@@ -0,0 +1,217 @@
...
+
+Successful platform engineering teams are characterized by a strong customer focus, empathy for the needs of application teams, and the ability to act as the "glue" that binds the organization together. Key roles and responsibilities within the platform team include:
+
+- **Customer-facing roles**: Serve as the primary point of contact for application teams, understanding their needs and advocating for their requirements within the platform.
+- **Infrastructure expertise**: Possess deep knowledge of the underlying infrastructure, whether on-premises or in the cloud, to ensure the platform is reliable and scalable.
+- **DevOps and automation skills**: Leverage infrastructure as code (IaC) tools and techniques to automate the provisioning and management of the platform.
+- **FinOps](https://www.pulumi.com/blog/finops-with-pulumi/) and cost optimization expertise**: Understand the organization's financial systems and processes to ensure the platform is cost-effective and aligned with budgetary constraints.
```
Comment (SaraDPH) [inline content/blog/the-guide-platform-engineering-idp-steps-best-practices/index.md:99]:
```suggestion
- **[FinOps](https://www.pulumi.com/blog/finops-with-pulumi/) and cost optimization expertise**: Understand the organization's financial systems and processes to ensure the platform is cost-effective and aligned with budgetary constraints.
```

**Pull request merged**